engine/src/main/java/org/hibernate/search/engine/search/dsl/predicate/MatchPredicateFieldSetContext.java

```diff
@@ -40,42 +42,35 @@ default MatchPredicateFieldSetContext orField(String absoluteFieldPath) {
 	MatchPredicateFieldSetContext orFields(String ... absoluteFieldPaths);
 
 	/**
-	 * Target the given <strong>raw</strong> field in the match predicate
-	 * as an alternative to the already-targeted fields.
+	 * Require at least one of the targeted fields to match the given value.
 	 * <p>
-	 * See {@link MatchPredicateContext#onRawField(String)} for more information about targeting raw fields.
+	 * This method will apply DSL converters to {@code value} before Hibernate Search attempts to interpret it as a field value.
+	 * See {@link DslConverter#ENABLED}.
 	 *
-	 * @param absoluteFieldPath The absolute path (from the document root) of the targeted field.
-	 * @return {@code this}, for method chaining.
+	 * @param value The value to match.
+	 * The signature of this method defines this parameter as an {@link Object},
+	 * but a specific type is expected depending on the targeted field.
+	 * See {@link DslConverter#ENABLED} for more information.
+	 * @return A context allowing to set further options or get the resulting predicate.
 	 *
-	 * @see MatchPredicateContext#onRawField(String)
+	 * @see #matching(Object, DslConverter)
 	 */
-	default MatchPredicateFieldSetContext orRawField(String absoluteFieldPath) {
-		return orRawFields( absoluteFieldPath );
+	default MatchPredicateTerminalContext matching(Object value) {
+		return matching( value, DslConverter.ENABLED );
 	}
 
-	/**
-	 * Target the given <strong>raw</strong> fields in the match predicate,
-	 * as an alternative to the already-targeted fields.
-	 * <p>
-	 * See {@link MatchPredicateContext#onRawFields(String...)} for more information about targeting raw fields.
-	 *
-	 * @param absoluteFieldPaths The absolute paths (from the document root) of the targeted fields.
-	 * @return {@code this}, for method chaining.
-	 *
-	 * @see MatchPredicateContext#onRawFields(String...)
-	 */
-	MatchPredicateFieldSetContext orRawFields(String... absoluteFieldPaths);
-
 	/**
 	 * Require at least one of the targeted fields to match the given value.
 	 *
 	 * @param value The value to match.
 	 * The signature of this method defines this parameter as an {@link Object},
-	 * but a specific type is expected depending on the targeted field.
-	 * See <a href="SearchPredicateFactoryContext.html#commonconcepts-parametertype">there</a> for more information.
-	 * @return A context allowing to set options or get the resulting predicate.
+	 * but a specific type is expected depending on the targeted field and on the {@code dslConverter} parameter.
+	 * See {@link DslConverter} for more information.
+	 * @param dslConverter Controls how the {@code value} should be converted before Hibernate Search attempts to interpret it as a field value.
+	 * See {@link DslConverter} for more information.
```

Comment:
Same here. I would avoid to repeat this line two times, one for each parameter.
I would move it to the end of the javadoc using a @see.